lib/Mojolicious/Plugin/OpenAPI/Security.pm

```diff
@@ -26,14 +30,14 @@ sub _build_action {
     my $security_completed = sub {
       my ($i, $status, @errors) = (0, 401);
 
-    SECURITY_AND:
+      SECURITY_AND:
```

Comment:
I don't understand why there's a diff here and in the rest of this file.

Reply:
Fixed with perltidy